frontend/appflowy_tauri/src/appflowy_app/components/editor/components/blocks/page/Page.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { forwardRef, memo } from 'react';
+import { EditorElementProps, PageNode } from '$app/application/document/document.types';
+
+export const Page = memo(
```

Comment:
From OpenAI suggestions:

Destructuring className: Directly destructuring className from the props for clarity. This makes it explicit that you're expecting a className prop and plan to use it.

Combined Class Name: Created a combinedClassName variable to handle the concatenation of class names. This improves readability and makes it easier to manage the class names.

Removed Unused Variable: The _node variable was unused in the original component. If it's not being used, it's better to omit it to avoid confusion.

```ts
export const Page = memo(
  forwardRef<HTMLDivElement, EditorElementProps<PageNode>>(({ node, children, className, ...attributes }, ref) => {
    const combinedClassName = `mb-2 text-4xl font-bold ${className ?? ''}`;

    return (
      <div ref={ref} {...attributes} className={combinedClassName}>
        {children}
      </div>
    );
  })
);
```